homeassistant/components/zwave_js/__init__.py

```diff
@@ -250,6 +291,18 @@ async def handle_ha_shutdown(event: Event) -> None:
             "node removed", lambda event: async_on_node_removed(event["node"])
         )
 
+        async def async_update_log_config(call: ServiceCall) -> None:
+            """Handle service call to update log config."""
+            await client.driver.async_update_log_config(LogConfig(**call.data))
+
+        # Register update_log_config service and pass service d
+        hass.services.async_register(
```

Comment:
Should this be a service or a websocket command exposed in the config panel? Will users automate the log level, or just change it to debug?

Reply:
Oh I didn't think of that but that's a better idea.

Reply:
Do we need to expose `log_to_file` and `filename` or should we keep it simple and just focus on setting log level?

Reply:
also I don't believe there is a way to read the current value for each of these attributes

Reply:
https://github.com/zwave-js/node-zwave-js/pull/1770

Reply:
We'd like to allow the user to capture the Z-Wave JS log at debug level with a button next to state dump in integration config panel. That would be the end goal I think.

Ie, we want to make it easy for users to get the Z-Wave JS log from the integration to help debug issues.

Reply:
I think that can be orchestrated with the exposed options. Basically, first call sets log to debug level and saves to file, second call resets